pkg/controller/controller.go

```diff
@@ -97,16 +97,40 @@ func (c *Controller) initOperatorConfig() {
 		c.logger.Infoln("no ConfigMap specified. Loading default values")
 	}
 
-	// env var takes priority over the same param from the operator ConfigMap
-	watchedNamespace := os.Getenv("WATCHED_NAMESPACE")
-	if watchedNamespace != "" {
-		c.logger.Infof("Watch the %q namespace specified in the env variable WATCHED_NAMESPACE\n", watchedNamespace)
-		configMapData["watched_namespace"] = watchedNamespace
-	}
+	watchedNsConfigMapVar, isPresentInOperatorConfigMap := configMapData["watched_namespace"]
+	watchedNsEnvVar, isPresentInOperatorEnv := os.LookupEnv("WATCHED_NAMESPACE")
```

Comment:
we don't need LookupEnv here, since empty or missing WATCHED_NAMESPACE should produce the same default behavior of watching the operator namespace.